### DISCUSSION TITLE
Swell calculates cart incorrectly and charges LESS than it should

### DISCUSSION THREAD

**Discussion by mewa** [swellstores/community#484] [Help] [answered]
I feel like I've raised several flavours of this issue already. However, it's been months (IIRC, I first raised a similar issue in August last year) and **Swell still calculates wrong amounts for customer carts** and makes us lose money as a result.

Example order below:
![2024-01-26-230253_989x297_scrot](https://github.com/swellstores/community/assets/7347194/91d9be61-7d38-4668-83b1-c57f8ea0cf67)

Shipping option is configured as follows:
![2024-01-26-230327_1114x269_scrot](https://github.com/swellstores/community/assets/7347194/2bffa50d-8001-4d7b-88c1-3adbee07442b)

How is this possible?

**Comment by steven-k1** [comment]
@mewa Can you please provide the store ID and order number from the example provided?

**Reply by mewa** [reply]
Sure, it's `wasabi-hq-pl` and the order number is `P100131`.

**Reply by steven-k1** [reply]
@mewa Thanks. Taking a look

**Reply by steven-k1** [reply, marked as answer]
@mewa Found that there was a recent issue causing shipping rules not to apply, which looks to be what occurred with order P100131. A fix was deployed yesterday, and shipping rules should be functioning properly again. 

Looking over your most recent order, I can see that the shipping was applied correctly.

**Comment by TheKayThatWasOrange** [comment]
If you didn't want to lose money then you shouldn't have chosen Swell. That's literally all they do.
